app/controllers/tracks_controller.rb

```diff
@@ -11,6 +11,6 @@ def index
   private
 
   def ordered_decorated_courses(track_courses)
```

Comment:
Why do we need to pass the `track_courses` as an argument?

Is there any reason not to use `@track.courses` inside `ordered_decorated_courses` method?